setup.py

```diff
@@ -29,6 +29,7 @@
         "itemadapter>=0.8.0",
         "Jinja2>=2.7",
         "price-parser>=0.3.4",
+        "scrapy>=2.0.1",
```

Comment:
@Gallaecio @PyExplorer I think we should keep this dependency optional. This looks easy to achieve, just by moving imports to methods which need them.

Reply:
OK, then I think we should add `extra` and `min-extra` tox environments that run with Scrapy installed.